dek-registry/src/main/java/io/confluent/dekregistry/web/rest/resources/DekRegistryResource.java

```diff
@@ -331,6 +331,43 @@ public void createKek(
     }
   }
 
+  @POST
+  @Path("/{name}/test")
```

Comment:
NIT: do you think we should name this `validate` instead of `test`. I think BYOK might call this process "validating"

Reply:
Good point, let me change